recipes-kernel/linux/linux-nilrt.inc

```diff
@@ -1,10 +1,18 @@
 inherit kernel
 require recipes-kernel/linux/linux-yocto.inc
 
+KBRANCH = "nilrt/${NI_RELEASE_VERSION}/${LINUX_VERSION}"
+GIT_KERNEL_REPO = "linux.git"
+
 machine_srcrev="${SRCREV}"
 
+# Subfolder of the same name will be added to FILESEXTRAPATHS and also
+# used for nilrt-specific config fragment manipulation during build.
+# Provide a unique name for each recipe saved in the same source folder
+KBUILD_FRAGMENTS_LOCATION ?= "nilrt"
+
 # Set the version for traceability purposes
-LINUX_VERSION_EXTENSION = "$([ -z "${BUILDNAME}" ] || echo '-')${BUILDNAME}"
+LINUX_VERSION_EXTENSION = "$([ "${LINUX_KERNEL_TYPE}" != "standard" ] && echo '-${LINUX_KERNEL_TYPE}')$([ -z "${BUILDNAME}" ] || echo '-')${BUILDNAME}"
```

Comment:
Does the `LINUX_KERNEL_TYPE` get set to `standard` in the "current" kernel recipe? I don't see that happening anywhere.

Reply:
meta-nilrt/recipes-kernel/linux/linux-nilrt.inc (which is used by all nilrt kernel recipes) includes openembedded-core/meta/recipes-kernel/linux/linux-yocto.inc which sets:

```
# The distro or local.conf should set this, but if nobody cares...
LINUX_KERNEL_TYPE ??= "standard"
```
We've been using this default for virtual/kernel package builds. There might be an argument to be made here about setting the (weak defined) default in distro or local.conf as the comment in that .inc file implies but we've been operating using the linux-yocto.inc default for a while so I didn't mess with it.

Reply:
OK. That's sufficient, I think.